nautobot/utilities/tables.py

```diff
@@ -41,6 +41,16 @@ def __init__(self, *args, user=None, **kwargs):
         for cpf in ComputedField.objects.filter(content_type=obj_type):
             self.base_columns[f"cpf_{cpf.slug}"] = ComputedFieldColumn(cpf)
 
+        for relationship in Relationship.objects.filter(Q(source_type=obj_type)):
+            self.base_columns[f"cr_{relationship.slug}"] = RelationshipColumn(
+                relationship, side=RelationshipSideChoices.SIDE_SOURCE
+            )
+
+        for relationship in Relationship.objects.filter(Q(destination_type=obj_type)):
+            self.base_columns[f"cr_{relationship.slug}"] = RelationshipColumn(
```

Comment:
A couple of gaps here:

- for non-symmetric same-type-to-same-type (e.g. Device-to-Device) relationships, you probably want to append `_src` or `_dst` to the key here because you want both columns to be visible; right now they'll clobber one another with the same key and you'll only see the destination column
- for symmetric relationships, you *do* want only a single column to be visible but it needs to include objects from either side of the relationship.